python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1058,6 +1066,8 @@ def execute_asset_backfill_iteration(
     else:
         check.failed(f"Unexpected backfill status: {backfill.status}")
 
+    tick_context.update_state(TickStatus.SUCCESS)
```

Comment:
to figure out: if the tick doesn't request any runs, do we want to consider that a SKIP?